CleverPush/CleverPush/Source/CleverPush.m

```diff
@@ -1462,21 +1464,65 @@ + (void)handleJSONNSURLResponse:(NSURLResponse*) response data:(NSData*) data er
     }
 }
 
++ (void)addSubscriptionTags:(NSArray*)tagIds {
+    [self addSubscriptionTags:tagIds callback:nil];
+}
+
++ (void)removeSubscriptionTags:(NSArray*)tagIds {
+    [self removeSubscriptionTags:tagIds callback:nil];
+}
+
++ (void)addSubscriptionTags:(NSArray*)tagIds callback:(void(^)(NSArray *))callback {
+    dispatch_group_t group = dispatch_group_create();
```

Comment:
What happens if I call this method in the main thread?
Will it block?

Reply:
Nope it will not block.